pkg/agent/util/net_windows.go

```diff
@@ -723,35 +745,35 @@ func NewNetNat(netNatName string, subnetCIDR *net.IPNet) error {
 	return nil
 }
 
-func ReplaceNetNatStaticMapping(netNatName string, externalIPAddr string, externalPort uint16, internalIPAddr string, internalPort uint16, proto string) error {
-	staticMappingStr, err := GetNetNatStaticMapping(netNatName, externalIPAddr, externalPort, proto)
+func ReplaceNetNatStaticMapping(mapping *NetNatStaticMapping) error {
+	staticMappingStr, err := GetNetNatStaticMapping(mapping)
 	if err != nil {
 		return err
 	}
 	parsed := parseGetNetCmdResult(staticMappingStr, 6)
 	if len(parsed) > 0 {
 		items := parsed[0]
-		if items[4] == internalIPAddr && items[5] == strconv.Itoa(int(internalPort)) {
+		if items[4] == mapping.InternalIP.String() && items[5] == strconv.Itoa(int(mapping.ExternalPort)) {
```

Comment:
Is this an intentional change? The previous code used internalPort but it uses externalPort

Reply:
I'll change it back.